### DISCUSSION TITLE
Left-side menu

### DISCUSSION THREAD

**Discussion by xmp00** [GitbookIO/community#257] [Feature Requests]
Hi,
I would love to have an option to control left-side menu to be sure that context that get's added doesn't gets schrinked. 
Either there should be an option and lef-side menu should auto-adjust depending on the content because otherwise it looks squeezed and ugly when you have a lot of papers to add.

Thank you

**Comment by addisonschultz** [comment]
Thanks for raising this - I was curious, do you have a link to one of the published sites you're running into this with? Would love to see the amount of pages you're talking to!
